### DISCUSSION TITLE
Interested in espnet-onnx ebranchformer ASR recipe. Is there a git branch I can pull from ?

### DISCUSSION THREAD

**Discussion by deblinbagchi** [espnet/espnet_onnx#105] [General]
Hello there ! Is anyone working on an ASR recipe involving espnet's ebranchformer ? Is there a feature branch I can pull from ? Please let me know. Thanks !

**Comment by Masao-Someki** [comment]
Hi @deblinbagchi, if you are using the latest version of `espnet` & `espnet-onnx`, then you can simply use the `ASRModelExport` class to export the e-branchformer model!
